manager.py

```diff
@@ -1,11 +1,12 @@
 # -*- coding: utf-8 -*-
-import os
-from flask import Flask, render_template, redirect, request
```

Comment:
I used Pycharm and It works fine.
But I don't know why it works fine and Pycharm remove 'render_template'.
I'll find the reason.